sunpy/database/attrs.py

```diff
@@ -196,7 +196,7 @@ def _create(wlk, root, session):
             path, inverted = value
             if inverted:
                 query = query.filter(or_(
-                    DatabaseEntry.path != path, DatabaseEntry.path == None))
+                    DatabaseEntry.path != path, DatabaseEntry.path is None))
```

Comment:
@cguennou we got this wrong... the `==` was not wrong as this is to construct a database query and not a simple variable comparison within python.  I've made you a PR (cguennou/sunpy#1) that hopefully fix it so pylint doesn't complain...